src/config.ts

```diff
@@ -2840,4 +2846,41 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/gravity-bridge/txs/{txHash}',
 	},
+	{
+		rpc: 'https://rpc.assetmantle.one/',
+		rest: 'https://rest.assetmantle.one/',
+		chainId: 'mantle-1',
+		chainName: 'AssetMantle',
+		stakeCurrency: {
+			coinDenom: 'MNTL',
+			coinMinimalDenom: 'umntl',
+			coinDecimals: 6,
+			coinGeckoId: 'asset-mantle',
+			coinImageUrl: window.location.origin + '/public/assets/tokens/mantle.png',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('mantle'),
```

Comment:
Please register 'mantle' on SLIP173
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Raised PR for the same
https://github.com/satoshilabs/slips/pull/1308